include/slb/type_traits.hpp

```diff
@@ -903,11 +903,63 @@ using add_pointer_t = typename slb::add_pointer<T>::type;
 // 23.15.7.6, other transformations
 
 using std::aligned_storage;
-using std::aligned_union;
 using std::decay;
 using std::enable_if;
 using std::conditional;
 
+#if !defined(__GLIBCXX__) || __has_include(<codecvt>) // >= libstdc++-5
```

Comment:
One more thing: can we move this whole block up to where `aligned_union` was before?

Reply:
Certainly! I thought we were putting pieces that we override at the bottom of their category.